src/Appwrite/Platform/Workers/Functions.php

```diff
@@ -92,6 +93,10 @@ public function action(Message $message, Database $dbForProject, Func $queueForF
             return;
         }
 
+        if ($function->isEmpty() && !empty($functionId)) {
+            $function = $dbForProject->getDocument('functions', $functionId);
```

Comment:
We could do `getDocument` when we added `setFunctionId()` and do `setFunction` instead - preventing need for this

Reply:
We don't have `dbForProject` injected the scheduler, only `dbForConsole`.